aat/src/aat/resources/features/F-106 New CategoryID Column/S-106.1.td.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This looks very similar to `Import_BEFTA_Master_Definition.td.json`

Reply:
Fixed in [6c59bbc](https://github.com/hmcts/ccd-definition-store-api/pull/1171/commits/6c59bbc4a4b15cfa2d8ed4a7d100b6b62fc34bc0)